user-guide/integrations/zapier-integration.md

```diff
@@ -0,0 +1,34 @@
+# Creating a Zap in Zapier for New Query Results
+
+You can connect Redash to Zapier and trigger an action every time a new line is added to you query results - for example, a Slack bot that announces every time a new order came in.
+The best way is to have a scheduled query that refreshes frequently so the notification you get is in real-time.
+
+For now you'll need to ask us for an invitation as it's still in beta - you can do that via Intercom (at the bottom right corner of every page in /help or inside the Redash webapp or [Slack](https://redash.slack.com/).
```

Comment:
1. The link to get an invite to Slack is slack.redash.io (they can't sign in from the other one).
2. Let's just post the Zapier link here... I checked again and didn't see anything explicitly prohibiting this :)